playground/doc/playground.mdx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import { mdx, MDXCreateElement } from '@mdx-js/react';
 import { Props, Description } from '~/props/dist/index';
-import { Playground } from '../dist';
+import { Playground, ShadowPlayground } from '../dist';
 import { Button } from '../test/index';
+import styles from '../test/styles.css';
 import { CssLayout } from '~/mdx-layout-css';
 export default CssLayout;
 
-# Playground Component
+## Playground Component
```

Comment:
I wasn't sure where to put ShadowPlayground, but creating a new one wasn't a good idea either because it's just a variation of this one

on the other hand the docs need to demo 2 components now
do we have some rules for such situations which I can follow or am I free to decide myself?

Reply:
@gqio can you please review this and maybe some other last changes?

Reply:
Feel free to decide yourself ;)